.github/workflows/test-pixi.yml

```diff
@@ -0,0 +1,32 @@
+name: Run tests with pixi
+
+on:
+  workflow_dispatch:
+  pull_request:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  # Execute a "nightly" build at 2 AM UTC
+  - cron:  '0 2 * * *'
```

Comment:
If you want to run a schedule job, could it make sense to delete the lock file for those? Otherwise we would be testing just GitHub Actions-images related regressions, that even if possible are not really frequent in my experience. See https://github.com/robotology/whole-body-estimators/blob/80d45bf79c5e4be3c7f83e0bec8f2cdfef48caab/.github/workflows/pixi-ci.yml#L30-L35 for example.

Reply:
Ok understood. Meanwhile, I'm thinking about just running these actions on pull requests, that is the original scope for this kind of action.